test/OpenTelemetry.Instrumentation.Http.Tests/HttpWebRequestTests.Basic.netfx.cs

```diff
@@ -95,6 +95,43 @@ public async Task HttpWebRequestInstrumentationInjectsHeadersAsync()
             parent.Stop();
         }
 
+        [Fact]
+        public async Task HttpWebRequestInstrumentationInjectsHeadersAsyncWhenActivityIsNotRecorded()
+        {
+            var activityProcessor = new Mock<BaseProcessor<Activity>>();
+            using var shutdownSignal = Sdk.CreateTracerProviderBuilder()
+                .AddProcessor(activityProcessor.Object)
+                .AddHttpWebRequestInstrumentation()
+                .Build();
+
+            var request = (HttpWebRequest)WebRequest.Create(this.url);
+
+            request.Method = "GET";
+
+            var parent = new Activity("parent")
+                .SetIdFormat(ActivityIdFormat.W3C)
+                .Start();
+            parent.TraceStateString = "k1=v1,k2=v2";
+            parent.ActivityTraceFlags = ActivityTraceFlags.None;
+
+            using var response = await request.GetResponseAsync();
+
+            Assert.Empty(activityProcessor.Invocations);
```

Comment:
how is this empty?

Reply:
its best to add comment like:
By default parentbasedsampler is used.
In this case, the parent is the manually created parentactivity, which will have TraceFlags as None.
This causes child to be *not* created.

Reply:
added. thanks!